core/includes/theme.inc

```diff
@@ -591,7 +591,7 @@ function _theme_process_registry(&$cache, $name, $type, $theme, $path) {
 
       // Allow variable preprocessors for all theme hooks, whether the hook is
       // implemented as a template or as a function.
-      // Check for existing variable preprocessors. Ensure arrayness.
+      // Check for existing variable preprocessors.
```

Comment:
We could say "and valid" here, since that's what "ensuring arrayness" tried to say I believe.

Reply:
@klonos Are you suggesting to change it to the following?

> Check for existing and valid variable preprocessors.

Reply:
@BWPanda yes 👍🏼 ...either that, or something this:
> Check for existing variable preprocessors, and ensure they are in the expected format.